opentracing-vertx-web/src/main/java/io/opentracing/contrib/vertx/ext/web/TracingHandler.java

```diff
@@ -0,0 +1,118 @@
+package io.opentracing.contrib.vertx.ext.web;
+
+import io.opentracing.contrib.vertx.ext.web.WebSpanDecorator.StandardTags;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+import io.opentracing.Span;
+import io.opentracing.SpanContext;
+import io.opentracing.Tracer;
+import io.opentracing.propagation.Format;
+import io.opentracing.tag.Tags;
+import io.vertx.core.Handler;
+import io.vertx.core.logging.Logger;
+import io.vertx.core.logging.LoggerFactory;
+import io.vertx.ext.web.RoutingContext;
+
+/**
+ * Handler which creates tracing data for all server requests. It should be added to
+ * {@link io.vertx.ext.web.Route#handler(Handler)} and {@link io.vertx.ext.web.Route#failureHandler(Handler)} as the
+ * first in the chain.
+ *
+ * @author Pavol Loffay
+ */
+public class TracingHandler implements Handler<RoutingContext> {
+    private static final Logger log = LoggerFactory.getLogger(TracingHandler.class);
+    public static final String ACTIVE_SEVER_SPAN = TracingHandler.class.getName() + ".severSpan";
```

Comment:
Might be better to use `CURRENT_SPAN` - to avoid expectation of it being an `ActiveSpan`.